include/curl/curl.h

```diff
@@ -2105,6 +2130,12 @@ typedef enum {
      this option is used only if PROXY_SSL_VERIFYPEER is true */
   CURLOPT(CURLOPT_PROXY_CAINFO_BLOB, CURLOPTTYPE_BLOB, 310),
 
+  /* SSL session cache callback. After the cache has been initialized,
+     allow user code to load SSL sesisonid-s to cache, only for OpenSSL.
+     The function must match the curl_ssl_cache_callback prototype. */
+  CURLOPT(CURLOPT_SSL_CACHE_FUNCTION, CURLOPTTYPE_FUNCTIONPOINT, 311),
+  CURLOPT(CURLOPT_SSL_CACHE_DATA, CURLOPTTYPE_CBPOINT, 312),
```

Comment:
`CURLOPT_SSL_CACHE_FUNCTION/DATA` are similarly not really saying what they do.

Reply:
This also mimicked `SSL_CTX` symbols, but sure, they'll be easy to add.